src/Tizen.NUI/src/internal/WebView/WebHttpRequestInterceptor.cs

```diff
@@ -44,6 +44,18 @@ protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef
             Interop.WebHttpRequestInterceptorPtr.DeleteWebHttpRequestInterceptorPtr(swigCPtr);
         }
 
+        /// <summary>
+        /// Gets webview in which request is intercepted.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public BaseComponents.WebView InterceptedWebView
+        {
+            get
+            {
+                return new BaseComponents.WebView(Interop.WebHttpRequestInterceptor.GetWebView(interceptorHandle), true);
```

Comment:
Could you check we can release the handle that get from `Interop.WebHttpRequestInterceptor.GetWebView(interceptorHandle)`?

And I hope this API returing a created C# object instead a new object
```suggestion
                return Registry.GetManagedBaseHandleFromNativePtr(Interop.WebHttpRequestInterceptor.GetWebView(interceptorHandle)) as BaseComponents.WebView ??
                 new new BaseComponents.WebView(Interop.WebHttpRequestInterceptor.GetWebView(interceptorHandle), true);
```

Reply:
@huayongxu could you please check this comment? I think it is good for avoiding unnecessary object creations.

Reply:
In BaseHandle.cs, == operator is overloaded:
public static bool operator ==(BaseHandle x, BaseHandle y)
Although InterceptedWebView is created newly, its equality with another WebView can be checked like:
if (interceptor.InterceptedWebview == webview)

Reply:
Ok, I understand